gulp-tasks/utils/async-to-generator.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable */
+// This is a light wrapper on top of the babel-helpers runtime code, meant to
+// be included in the workbox-core bundle.
+var babelHelpers = {
```

Comment:
I'd suggest assigning this to `self.babelHelpers`. That makes it more clear that this is intended to be used outside of this module.

Reply:
I'd also suggest linking to the source of this file on Github.

If there's no easy way to dynamically include this (I looked for a way and couldn't find an obvious solution), then I think the second best thing is to add a comment in the source as to where it came from and suggest periodically looking to see if it's been updated.

Reply:
Oh, one more thing...

I think we should rename this to `external-helpers.js` or something like that and add a comment explaining why this is the only helper we're including.

I can see a situation where we need to add more helpers in the future, and we might not remember why we only included this one.